packages/api/src/routes/medical/dtos/organizationDTO.ts

```diff
@@ -7,13 +7,15 @@ export type OrganizationDTO = BaseDTO & {
   name: string;
   type: OrgType;
   location: AddressStrictDTO;
+  businessType: OrganizationBizType;
 };
 
 export function dtoFromModel(org: Organization): OrganizationDTO {
   const { name, type, location } = org.data;
   return {
     ...toBaseDTO(org),
     oid: org.oid,
+    businessType: org.type,
```

Comment:
ditto idk think we want to expose this to cxs

Reply:
Moved to the /cx-data endpoint.